Source/WebKit/UIProcess/API/APIPageConfiguration.cpp

```diff
@@ -100,6 +100,21 @@ BrowsingContextGroup& PageConfiguration::browsingContextGroup() const
     return m_data.browsingContextGroup.get();
 }
 
+void PageConfiguration::setBrowsingContextGroup(BrowsingContextGroup& group)
```

Comment:
Can this take a `RefPtr<>&&` like the other ones in this file?

Reply:
Sure